zeppelin-web/src/app/jobmanager/jobs/job.controller.js

```diff
@@ -38,6 +38,68 @@
       return isNaN(result) ? 0 : result;
     };
 
+    $scope.runNotebookJob = function(notebookId) {
+      BootstrapDialog.confirm({
+        closable: true,
+        title: '',
+        message: 'Run all paragraphs?',
+        callback: function(result) {
+          if (result) {
+            $http({
+              method: 'POST',
+              url: baseUrlSrv.getRestApiBase() + '/notebook/job/' + notebookId,
+              headers: {
+                'Content-Type': 'application/x-www-form-urlencoded'
+              }
+            }).then(function successCallback(response) {
+              // success
+            }, function errorCallback(errorResponse) {
+              var errorText = 'SERVER ERROR';
+              if (errorResponse.data.message !== undefined) {
+                errorText = errorResponse.data.message;
+              }
+              BootstrapDialog.alert({
+                closable: true,
+                title: 'Execution Failure',
+                message: errorText
+              });
+            });
+          }
+        }
+      });
+    };
+
+    $scope.stopNotebookJob = function(notebookId) {
+      BootstrapDialog.confirm({
+        closable: true,
+        title: '',
+        message: 'Stop all paragraphs?',
+        callback: function(result) {
+          if (result === true) {
```

Comment:
@cloverhearts thanks for addressing my comment. however, you seemed to ignore the "elsewhere" part of it. could you fix it here as well?